libraries/AC_WPNav/AC_WPNav.cpp

```diff
@@ -93,6 +93,15 @@ const AP_Param::GroupInfo AC_WPNav::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("TER_MARGIN",  12, AC_WPNav, _terrain_margin, 10.0),
 
+    // @Param: ACCEL_C
+    // @DisplayName: Waypoint Cornering Acceleration
+    // @Description: Defines the maximum cornering acceleration in cm/s/s used during missions, zero uses max lean angle.
+    // @Units: cm/s/s
```

Comment:
Could be in meters?  The other accels are in cm, but jerk is in meters...

Reply:
Yeh, I saw that too. I would prefer meters but it is hard to make it different to the others. We have to change all these anyway. I suspect we will have to change the whole family name or something.